package.json

```diff
@@ -31,6 +31,7 @@
     "axios": "^0.26.1",
     "core-js": "^3.6.5",
     "express": "4.17.2",
+    "lodash-es": "^4.17.21",
```

Comment:
`lodash-es` so we can use tree shaking to reduce builds. If we used the original `lodash` package we would have to do `import debounce from 'lodash/debounce'` but it's not inherently obvious that `import {debounce} from 'lodash` would import the whole library. This is why `lodash-es` is hand, we can do `import {debounce} from 'lodash` without any worries